_gsdocs-proposals/2020/proposal_AllpixSquaredRestructureDocs.md

```diff
@@ -0,0 +1,57 @@
+
+The documentation of our project consists of several parts, which live in different spaces as detailed below:
+
+* **Website:** The first point of contact should be the website. It is built using the static website generator `hugo` based on the very simple `beautiful-hugo` theme and contains, most importantly, the "news" section used for publishing release announcements and release notes. Furthermore, some installation information (mostly redundant to the user manual) and screenshots illustrating different use cases are provided.
+* **User Manual:** The main documentation of the framework consists of a 160-page user manual document which is generated from LaTeX code in the main software repository as well as Markdown files for individual parts of the framework - translated to LaTeX using `pandoc`. This happens automatically for new releases and it is published
+    * Online as HTML version with a link from the website (using `latex2html`)
+    * As PDF document, linked from the website (using `pdflatex`)
+    
+* **Doxygen Code Reference:** In addition to the user manual, also the source code is heavily documented and a HTML documentation of the code base is automatically generated using `doxygen` and deployed on the website.
+
+With over 3.5 years of continuous development and additions, the documentation has become a bit cluttered and we would like to streamline it with your help:
+
+* **Content:** We would like to separate more clearly which parts of the documentation are intended for users which are not expected to alter a single line of code - and which parts aim at developers who would like to contribute to the project or would like to extend their own copy. Also, we would like to review the current user manual for potentially lacking information or areas where we could improve.
+* **Infrastructure:** We would like to streamline the way we present the documentation and especially improve its online representation. Optimally we would end up with a streamlined, fully-integrated online presence which coherently presents news, manual and code reference. One possibility for this would e.g. be moving the documentation to [Docsy.dev](https://www.docsy.dev/). The challenging part about this is that we would like to keep as much as possible of our current automatic deployment and workflow.
+* **Bonus task:** For those of you who would like to dive a bit deeper into the scientific depth of this project - we currently do have a few examples for users to understand how to start using the framework and how to use certain features, but they are not very nicely presented and not very extensively documented. They could use a good polishing. Ultimately, these and further examples can be streamlined to follow a tutorial-like workflow.
```

Comment:
Just a suggestion, part of your bonus task or something similar could be used as exercise to give to candidates, allowing you to select them (in addition to their portfolio).

Reply:
Is this something we need to state here or is it a bit like GSoC that we just send the evaluation task to the applicants who are interested?

Reply:
No, it was just a comment, no need to mention it in the proposal. Resolving conversation.